packages/browser/src/Component/Widget/Server/SPARQL/Readme.md

```diff
@@ -108,6 +78,32 @@ The JSON view returns a collapsible representation of the query reponse.
 
 ![JSON view demonstraction](/docs/static/Widget/Extensions/SPARQL/Pictures/sparql-widget-json-demo.gif)
 
+### Widget Configuration
+
+The minimal configuration required to make the SPARQL module work is the following :
+
+```json
+{
+  "sparqlModule": {
```

Comment:
config should be splitted

Reply:
added to assets/config/widget/widget/sparql_widget.json